verification/README.md

```diff
@@ -0,0 +1,248 @@
+# <h1 align="center"> Verification </h1>
+
+---
+
+A contract verification service. Runs as an HTTP server and allows
+making verification requests through REST API. It is stateless
+and answers requests based on provided information only.
+
+## Building from source
+Install rustup from rustup.rs.
+```
+git clone git@github.com:blockscout/blockscout-rs.git
+
+cd blockscout-rs
+
+cargo build --all --release
+```
+You can find the built binary in `target/release` folder.
+
+## Installing through cargo
+Another way to install the binary without cloning the repository is to use cargo straightway:
+```
+cargo install --git https://github.com/blockscout/blockscout-rs --bin verification
+```
+In that case, you can run the binary using just `verification`.
+
+## Configuration
+Currently, the service uses a configuration file the path to which is specified via CLI flag `--config-path=[path]`.
+The configuration file may contain the following options:
+```toml
+[server]
+# IP address and port number the server should listen to
+addr = "0.0.0.0:8043"
+
+[solidity]
+# if false solidity verification is not supported
```

Comment:
`enables solidity verification. when disabled, will not load solidity required data, like compilers list`

Reply:
I believe that it is better not to say `enables solidity verification` as it actual a default behavior. The flag mainly disables solidity verification, doesn't it?